oshc/main/templates/session.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %} {% load static %} {% block content %}
+
+<iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://vaibhavsingh.typeform.com/to/Y4ioHZ"></iframe>
```

Comment:
I think the URL should be something like `oshelp.typeform.com` or `opensourcehelp.typeform.com` so that the same Typeform account can be used in future._[cosmetic change]_